samples/update_protection_runs/README.md

```diff
@@ -0,0 +1,137 @@
+## Usage 
+```
+python --job_name "MSSQL-protect us-west-1" --start_time 2019-11-1 --end_time
+ 2019-11-18 --extend 10
+```
+
+## Connect to the Cohesity Cluster
+First make sure that you are connected to a Cohesity Cluster.
+```
+cohesity_client = CohesityClient(cluster_vip=CLUSTER_VIP,
+                                 username=CLUSTER_USERNAME, 
+                                 password=CLUSTER_PASSWORD,
+                                 domain=DOMAIN)
+```
+## Example
+``` 
+ body.job_runs.append(UpdateProtectionJobRun())
```

Comment:
We can have a short but complete example function, to show implementation.

Reply:
Hmm, readme should have actual body/payload, this is just to copy paste the code if the customers going through the examples are in doubt.